core/model/modx/processors/system/config_check.inc.php

```diff
@@ -91,7 +91,11 @@ function getResourceBypass(modX &$modx,$criteria) {
         curl_setopt($curl, CURLOPT_VERBOSE, false);
         /* follow rewrites, e.g. http to https rewrites */
         /* if a sites rewrites the checkurl to 200 page, this method fails */
-        curl_setopt($curl, CURLOPT_FOLLOWLOCATION, true);
+        $safeMode = @ini_get('safe_mode');
```

Comment:
Because ini_get can be disabled by security reasons and we will receive 2 additional warnings instead of one.

> Using @ turns off all optimization in the whole file

Realy? Than why I see no difference if trying to log time after this commands width and without @?
```
[2015-10-31 12:56:40] (ERROR @ /manager/index.php) Without supression: 0.082990169525146
[2015-10-31 12:56:43] (ERROR @ /manager/index.php) Without supression: 0.073740005493164
[2015-10-31 12:56:46] (ERROR @ /manager/index.php) Without supression: 0.11430478096008
[2015-10-31 12:56:50] (ERROR @ /manager/index.php) Without supression: 0.072605133056641
[2015-10-31 12:56:53] (ERROR @ /manager/index.php) Without supression: 0.067651987075806
[2015-10-31 12:57:08] (ERROR @ /manager/index.php) With supression: 0.085567951202393
[2015-10-31 12:57:13] (ERROR @ /manager/index.php) With supression: 0.06732702255249
[2015-10-31 12:57:17] (ERROR @ /manager/index.php) With supression: 0.08738899230957
[2015-10-31 12:57:20] (ERROR @ /manager/index.php) With supression: 0.066944122314453
[2015-10-31 12:57:23] (ERROR @ /manager/index.php) With supression: 0.075565814971924
```
You can check it yourself - just add
`
$this->modx->log(1, 'With supression: ' . (microtime(true) - $this->modx->startTime));
` clear cache and refresh manager main page for a while